cmd/finch/virtual_machine_status_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/runfinch/finch/pkg/dependency"
+	"github.com/runfinch/finch/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewStatusVMCommand(t *testing.T) {
+	t.Parallel()
+
+	cmd := newStatusVMCommand(nil, nil)
+	assert.Equal(t, cmd.Name(), "status")
+}
+
+func TestStatusVMAction_run(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
```

Comment:
Add a field for status output.

Reply:
Done that, but have to take a closer look into getting the stdout as you suggested above.

Reply:
Yeah this change is not pushed yet.